README.md

```diff
@@ -126,9 +126,9 @@ Below are examples of those components and description of the props they are acc
 |        `headerSize`         |                 `(h: LayoutRectangle) => void`                  |   Yes    |             Handler that is called when header's size changes              |
 |           `image`           |                      `ImageSourcePropType`                      |   Yes    |                                                                            |                                       Sets header image                                       |
 | `keyboardShouldPersistTaps` |             `boolean, "always", "never", "handled"`             |   Yes    |                                                                            |                               Set keyboard persist taps method                                |
-|        `leftTopIcon`        |                      `ImageSourcePropType`                      |   Yes    |                                                                            |                                 Set icon for left top button                                  |
+|        `leftTopIcon`        |           `() => ReactNode` &#124; `ImageSourcePropType`        |   Yes    |                                                                            |                                 Set icon for left top button                                  |
```

Comment:
Currently we have two readme files - one is in main repo file and other docs file are in `website` directory. Could you make sure that it's changed there too?